src/Symfony/Bundle/FrameworkBundle/DependencyInjection/Configuration.php

```diff
@@ -610,7 +611,6 @@ private function addTemplatingSection(ArrayNodeDefinition $rootNode)
                         ->then(function () { return ['enabled' => false, 'engines' => false]; })
                     ->end()
                     ->children()
-                        ->scalarNode('hinclude_default_template')->defaultNull()->end()
```

Comment:
Shouldn't we keep it and move it to the new option (with deprecation)?

Reply:
We do. Otherwise, that's a BC break.